### PR TITLE
Run Bdd tests on Travis CI

### PR DESCRIPTION
**Pull request by dpobel**
Come on Travis!

requires https://github.com/ezsystems/ezplatform/pull/95 to have a router for the behat environment and more independent shell scripts to initialize the environment.

### CODE REVIEW

**Comment by bdunogier** [comment]
Feel free to open a related PR to ezplatform with the router changes.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ matrix:
           env: BEFORE="./bin/travis/setupnode.sh" TEST_CMD="grunt test" AFTER_SUCCESS="./bin/travis/generate_apidoc.sh"
         - php: 7.0
           env: BEFORE="composer install --prefer-dist" TEST_CMD="./vendor/phpunit/phpunit/phpunit -c phpunit.xml" AFTER_SUCCESS='echo "After success"'
+        - php: 7.0
+          env: BEFORE="./bin/travis/prepare_behat.sh" EST_CMD="php bin/behat -vv --profile=platformui" AFTER_SUCCESS='echo "After success"'
```
Comment (bdunogier) [inline .travis.yml]: EST -> TEST ?

Comment (bdunogier) [inline .travis.yml]: Looks good, but it won't run behat because of EST (travis did not fail, but behat didn't really run).

Comment (dpobel) [inline .travis.yml]: TEST yes, Friday... ;-)

**Comment by bdunogier** [comment]
Fails because of `bin/behat` not being found. Try updating `composer.json` like this (in addition to whatever is there):

```
    "config": {
        "bin-dir": "bin"
    },
```

**Comment by dpobel** [comment]
ping @andrerom @bdunogier @miguelcleverti

**Comment by andrerom** [comment]
+1

**Comment by miguelcleverti** [comment]
+1

**Pull request merged**